package.json

```diff
@@ -43,7 +43,6 @@
     "@types/text-encoding": "^0.0.40",
     "@types/yup": "^0.29.11",
     "@uswds/uswds": "^3.12.0",
-    "apollo": "^2.34",
```

Comment:
Thanks for removing this -- context on this is I think since we cut over to the new apollo codegen package we don't need this anymore!